tests/integration/test_fim/test_files/test_restrict/test_restrict.py

```diff
@@ -0,0 +1,109 @@
+"""
+ Copyright (C) 2015-2024, Wazuh Inc.
+ Created by Wazuh, Inc. <info@wazuh.com>.
+ This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+"""
+
+import sys
+import time
+import pytest
+import os
+
+from pathlib import Path
+from wazuh_testing.constants.paths.logs import WAZUH_LOG_PATH
+from wazuh_testing.constants.platforms import WINDOWS
+from wazuh_testing.modules.agentd.configuration import AGENTD_DEBUG, AGENTD_WINDOWS_DEBUG
+from wazuh_testing.modules.fim.patterns import EVENT_TYPE_ADDED, FIM_EVENT_RESTRICT
+from wazuh_testing.modules.fim.utils import get_fim_event_data
+from wazuh_testing.modules.monitord.configuration import MONITORD_ROTATE_LOG
+from wazuh_testing.modules.fim import configuration
+from wazuh_testing.tools.monitors.file_monitor import FileMonitor
+from wazuh_testing.utils.callbacks import generate_callback
+from wazuh_testing.utils.configuration import get_test_cases_data, load_configuration_template
+
+from . import TEST_CASES_PATH, CONFIGS_PATH
+
+# Pytest marks to run on any service type on linux or windows.
```

Comment:
Is this comment right? These tests are not marked to run on Windows, I don't know if that's OK.